### DISCUSSION TITLE
update.message.reply_text does not support markdown

### DISCUSSION THREAD

**Discussion by handsfreem** [python-telegram-bot/python-telegram-bot#3700] [Q&A] [answered]
Will this issue be resolved in the future?

**Comment by Bibo-Joshi** [comment, marked as answer]
Sure it does: `message.reply_text(text="*bold*", parse_mode=telegram.constants.ParseMode.MARKDOWN_V2)`.

**Comment by cal-cmd** [comment]
I'm having the issue here too with update.message.reply_text..
